### ISSUE TITLE
RBAC rules are insufficient: Failed to list *v1alpha1.SealedSecret

### ISSUE THREAD

**Issue by farcaller** [bitnami/sealed-secrets#29]
v0.3.0 release fails in 1.7 cluster:

```
E0703 13:41:25.346224       6 reflector.go:201] github.com/bitnami/sealed-secrets/cmd/controller/controller.go:126: Failed to list *v1alpha1.SealedSecret: User "system:serviceaccount:kube-system:sealed-secrets-controller" cannot list sealedsecrets.bitnami.com at the cluster scope. (get sealedsecrets.bitnami.com)
```

**Comment by farcaller** [comment]
Upon further inspection, the demo controller is lacking the binding to `secrets-unsealer` ClusterRole.

**Comment by anguslees** [comment]
Thanks for the report!

I will have to also investigate why this didn't show up during integration
testing...

**Comment by anguslees** [comment]
Fixed as part of the RBAC/no-RBAC split in #24
Just-released v0.3.1 contains the fix.

Thanks again for reporting the issue...
